src/Table/Column/FilterModel.php

```diff
@@ -65,6 +65,9 @@ public static function factoryType(App $app, Field $field): self
             Types::DATETIME_MUTABLE => FilterModel\TypeDatetime::class,
             Types::DATETIME_IMMUTABLE => FilterModel\TypeDatetime::class,
 
+            Types::JSON => FilterModel\TypeString::class,
```

Comment:
These types are casted to non-string in PHP, the php values are never used in these filters?

Reply:
Yes it looks like that. For example, type=json field shows as text in grid column and looks like filter works on top of it.
Probably @ibelar could check this in more details too.
Of course, filtering as string will not be the best option, but at least it will not throw exception then and at least partly work as filter.

Reply:
I also wonder if such filtering is even cross-DB compatible when the DB column type is native like JSON.

If it is, then maybe we can add `FilterModel\TypeString::class` as a fallback for unspecified type

Reply:
If not compatible with all DB engines, then maybe we can somehow not add filter on these columns at all (automatically)?
What I don't like there is that ugly exception we get if json column is added in Grid with filters.